crates/iota/src/name_commands.rs

```diff
@@ -1356,6 +1415,13 @@ impl IotaNamesNftProxy {
     }
 
     fn module_name(&self) -> &'static str {
+        match self {
+            IotaNamesNftProxy::Domain(_) => "controller",
```

Comment:
Tbh this feels weird, very tailored to solve this PR's issue

Reply:
Yeah, I know. But how would you rather do it?

Reply:
I think we need to either:
- resolve the module name when it's actually needed; or
- rename this function
`module_name` is too broad for this, maybe `controller_module_name` or something as `subdomain_proxy` essentially forward to the controller anyway

Reply:
Also applies to `package_id` though, I think I need to think a bit more about it, I don't like it